cypress/e2e/playground.cy.ts

```diff
@@ -0,0 +1,87 @@
+const getIframeBody = () => {
```

Comment:
Is this used anywhere?

Reply:
nope it was an artifact from the tests i was referencing